### DISCUSSION TITLE
Mobile app companion

### DISCUSSION THREAD

**Discussion by jowitte** [svsool/vscode-memo#533] [Q&A]
I'm a big fan of memo and us it to organize everything from Zettelkasten to meeting notes and project management. I use markdown editors on my iPad/iPhone to edit files which works somehow. 1Writer even follows [[links]], but only if the links points to a file in the same directory. I don't use a lot of directories and keep stuff flat, but it would really be helpful to have this configured and also have an app which actually can create links in memo style with suggestions. 

Anybody has a tip? I even tried running vscode on my own EC2 instance but that was overdose in terms of cost and features on my iPhone...

**Comment by svsool** [comment]
Hi @jowitte, one option could be to try Obsidian [mobile apps](https://obsidian.md/mobile), I haven't tried it though. I'd expect it to be less or more compatible. Let me know if works.

**Comment by jowitte** [comment]
Hey @svsool. Thanks for bringing up Obsidian. I was testing this a while ago and was unsatisfied with their limited sync support. I now managed to use it via iCloud and it seems like a _good enough_ companion. Thanks for support.
